tests/integration/components/user-profile-ics-test.js

```diff
@@ -43,6 +43,7 @@ module('Integration | Component | user profile ics', function(hooks) {
 
     await render(hbs`<UserProfileIcs @isManaging={{true}} @user={{this.user}} @setIsManaging={{noop}} />`);
     await click('.refresh-key');
+    await settled();
```

Comment:
I'm guessing our test harness awaits this chain of async operations.  But we must be missing something 🤔 

```
  @dropTask
  *refreshKey() {
    const token = yield this.randomToken(this.args.user.id);
    this.args.user.set('icsFeedKey', token);
    yield this.args.user.save();
    this.args.setIsManaging(false);
    this.hasSavedRecently = true;
    yield timeout(500);
    this.hasSavedRecently = false;
  }
```

Reply:
IIRC I pulled this `settled()` out a week or so ago since it  didn't seem to be needed anymore and tests ran, but I'm not fussed about it being back. Thanks!